diff --git a/src/containers.h b/src/containers.h
--- a/src/containers.h
+++ b/src/containers.h
@@ -16,7 +16,7 @@
 #include <net/if.h>
 #include <netinet/in.h>
 #if defined(__linux__)
-#include <linux/wireguard.h>
+#include "uapi/linux/linux/wireguard.h"
 #elif defined(__OpenBSD__)
 #include <net/if_wg.h>
 #endif
diff --git a/src/ipc-linux.h b/src/ipc-linux.h
--- a/src/ipc-linux.h
+++ b/src/ipc-linux.h
@@ -13,7 +13,7 @@
 #include <linux/if_link.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
-#include <linux/wireguard.h>
+#include "uapi/linux/linux/wireguard.h"
 #include <netinet/in.h>
 #include "containers.h"
 #include "encoding.h"
